### ISSUE TITLE
current c1 img is missing Update Manager and synaptic

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-core-releases#51]

**Comment by llelectronics** [comment]
Is missing currently as it depends on synaptic.

**Comment by star-buck** [comment]
Then we manually can add it?

**Comment by llelectronics** [comment]
Yes if you manually add synaptic you can install mintupdate from the repo

**Comment by star-buck** [comment]
how can we get an img that has synaptic and update manager installed, that doesnt uninstall them when updating and as soon as synaptic comes back into the official repo, it works normally?
I need such an img for C1 to ship.

**Comment by llelectronics** [comment]
I could write a live build hook that grabs the deb from debian unstable and installs it together as getting mint update again as a temporary workaround for the issue. (its the quickest workaround possible and easily done in contrast to cloning synaptics source and building it on dci which will land in our repositories then and could get a higher priority than the debian repos version when later appearing again)
The installed packages will only be affected by newer packages landing in the repositories. So if a newer version of Synaptic lands in buster it will get updated for our users.

**Comment by star-buck** [comment]
Lets do this then as temp. workaround.
We can aim for a 19.06 release for desktop and core versions once buster
becomes really stable.

**Comment by star-buck** [comment]
any success for it + new img?

**Comment by llelectronics** [comment]
There is a test image already here which uses the workaround script. Its a first build I did not test it yet. Will do this tomorrow: http://ds9-eu.s3-eu-central-1.amazonaws.com/dci/images/1901/armhf/netrunner-core/C1/netrunner-core-c1-20190410.1809-armhf.img.gz
